target/linux/generic/config-6.1

```diff
@@ -1954,6 +1954,7 @@ CONFIG_DVB_MAX_ADAPTERS=16
 # CONFIG_EEPROM_LEGACY is not set
 # CONFIG_EEPROM_MAX6875 is not set
 # CONFIG_EFI is not set
+CONFIG_EFI_HANDOVER_PROTOCOL=y
```

Comment:
This symbol is only for ARCH x86. I guess we need to move it to x86 sub-target.

Reply:
I moved it to `target/linux/x86/config-6.1`